soc/arm/st_stm32/stm32l1/soc.h

```diff
@@ -51,6 +51,10 @@
 #include <stm32l1xx_ll_i2c.h>
 #endif
 
+#ifdef CONFIG_SPI
```

Comment:
We usually use drivers Kconfig here (`CONFIG_SPI_STM32`), not directly the api/peripheral reference.

Reply:
Sure, see that in the other files.  Had just copied the CONFIG_I2C line above. Presumably that just slipped through.  will fix.